src/features/kickstart-adg.feature

```diff
@@ -8,12 +8,15 @@ Feature: Kickstart adg process, to source data and valid final tables for expect
     Given The template file 'record_template.json' as an input
     And Generate '10' records per table for 'vacancy' with PII flag as 'False' and upload to s3 bucket
     And Generate '10' records per table for 'application' with PII flag as 'True' and upload to s3 bucket
+#    And Generate '10' records per table for 'payment' with PII flag as 'True' and upload to s3 bucket
```

Comment:
We shouldn't be adding commented code to master. Should be either removed or uncommented.